src/components/Modal/Account.vue

```diff
@@ -71,10 +70,10 @@ watch(open, () => (step.value = null));
     </div>
     <div v-else>
       <div v-if="$auth.isAuthenticated.value" class="m-4 space-y-2">
-        <a
-          :href="explorerUrl(web3.network.key, web3.account)"
-          target="_blank"
+        <BaseLink
+          :link="explorerUrl(web3.network.key, web3.account)"
           class="block"
+          hide-external-icon
```

Comment:
Now that I think of it, I think by default the external icon should be hidden, so that this prop does not have to be passed everytime when we don't need it.

Reply:
It's about 50-50, I'd prefer to have the prop like this.

Reply:
I'd also say as per default external links should usually have such an indicator. Only if really not wanted for some layout/styling reasons it should be omitted.

Reply:
Awesome please approve if there isn't something else :)